G/gflags/build_tarballs.jl

```diff
@@ -0,0 +1,40 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "gflags"
+version = v"2.2.2"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/gflags/gflags.git", "986e8eed00ded8168ef4eaa6f925dc6be50b40fa")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir
+cd gflags/
+mkdir build
+cd build
+cmake -DCMAKE_INSTALL_PREFIX=$prefix -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} -DCMAKE_BUILD_TYPE=Release -DBUILD_SHARED_LIBS=ON ..
+make
```

Comment:
Minor changes to make the script slightly nicer:
```suggestion
cd $WORKSPACE/srcdir/gflags/
mkdir build
cd build
cmake -DCMAKE_INSTALL_PREFIX=${prefix} \
    -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} \
    -DCMAKE_BUILD_TYPE=Release \
    -DBUILD_SHARED_LIBS=ON \
    ..
make -j${nproc}
```